Add_Structure.ps1

```diff
@@ -102,7 +102,7 @@ Write-Progress -Activity $Progress_Activity -PercentComplete 1
 
 $Check_Sources_Files_Count = (Get-ChildItem -Path "$Current_Folder\Sources\Run_in_Sandbox" -Recurse).count
 
-if ($Check_Sources_Files_Count -ne 40) {
+if ($Check_Sources_Files_Count -lt 40) {

```

Comment:
Hm, i am not sure about this, but i´ll approve this now. I already through about changing the checking to be more precise and include a file list or something that is required.

Reply:
This was just my quick fix to prevent the script from erroring due to the additional files included in the source (when I had downloaded a few commits ago). I figured that having extra files being present isn't usually going to be an issue compared to missing files - so should be no harm in changing the condition to be less-than.

But as you say, an even better future improvement would be to validate for specific required files being present, rather than just going by the count.